components/ContentFlow.brs

```diff
@@ -218,7 +218,7 @@ sub onTruexAdDataReceived(event as object)
     tarInitAction = {
         type: "init",
         adParameters: {
-            vast_config_url: determineVastConfigUrl(decodedData.vast_config_url, decodedData.user_id),
+            vast_config_url: determineVastConfigUrl(decodedData.vast_config_url),
             placement_hash: decodedData.placement_hash
```

Comment:
I think we should consider configuring `logLevel`, `channelWidth` and `channelHeight` here, as an example?

Reply:
Yes, good idea, I added this.